bin/proving-service/README.md

```diff
@@ -9,7 +9,36 @@ A worker is a gRPC service that can receive transaction witnesses, proposed batc
 
 The proxy uses [Cloudflare's Pingora crate](https://crates.io/crates/pingora), which provides features to create a modular proxy. It is meant to handle multiple workers with a queue, assigning a worker to each request and retrying if the worker is not available. Further information about Pingora and its features can be found in the [official GitHub repository](https://github.com/cloudflare/pingora).
 
-## Installation
+
+## Debian Installation
+
+#### Prover
+```bash
+sudo wget https://github.com/0xMiden/miden-base/releases/download/v0.8/miden-prover-v0.8-arm64.deb -O prover.deb
```

Comment:
Might be worth adding a step that verifies the package hash with the sha256sum file that we produce, so that users are aware they can do so, but totally your call on that.

Reply:
Yea good call. I'll circle back to that once I get this stuff out on the infra

Reply:
Nvm I added it here https://github.com/0xMiden/miden-base/pull/1332/commits/4ce9cd5ba41b65c0c9d23c6d5e8487fe2adae04e

Reply:
Tested on ubuntu